src/lib/jwt.ts

```diff
@@ -47,5 +47,5 @@ export function getTokenDefaultIat() {
 }
 
 export function isTokenIatGreaterThanDate(date: Date, iat: number) {
-  return Math.floor(date.getTime() / 1000) < iat;
+  return Math.floor(date.getTime() / 1000) <= iat;
```

Comment:
Not considering the same date as valid is a bit troublesome. This will cause problems also with the "password change".

Reply:
If is a bit troublesome why are we doing it?